frontend/dispatcher.lua

```diff
@@ -332,11 +336,29 @@ function Dispatcher:init()
     for i=1,#CreOptions do
         parseoptions(CreOptions, i)
     end
+    UIManager:broadcastEvent(Event:new("DispatcherRegisterActions"))
     Dispatcher.initialized = true
 end
 
+--[[--
+    add settings at runtime
+    @param name: the key to use in the table
+    @param value: a table per settingsList above.
+    see helloworld plugin for an example.
```

Comment:
I believe this should go above the params separated by newlines. Also you can/should include a link.